src/booleans.js

```diff
@@ -1,61 +1,77 @@
 function negate(a) {
-  // your code here
-};
+  return !a;
+}
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b;
 };
 
 function none(a, b) {
-  // your code here
+  return !a && !b;
 };
 
 function one(a, b) {
-  // your code here
+  return (!a && b) || (a && !b);
 };
 
 function truthiness(a) {
-  // your code here
+  return !!a;
 };
 
 function isEqual(a, b) {
-  // your code here
+  return a === b;
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  return a>b;
```

Comment:
There should be a space between your variables and operands in your functions, a few more instances of this below
```
  return a > b;
```